packages/cadl-ranch-coverage-sdk/src/types.ts

```diff
@@ -33,6 +34,12 @@ export type ScenarioStatus =
   // Scenario is not applicable in current test.
   | "not-applicable";
 
+export type ScenarioType =
+  // Report for branded scenarios
+  | "branded"
```

Comment:
Compared with "branded", I prefer "azure".

Reply:
yeah I kinda agree, I don't really like that name for someone not in this team this is very confusing. I think we should just show all the different emitters by name instead and have a way to filter so you can just show the azure emitters or omit them

Reply:
This type is actually not used. Removed and changed all the places related to type to azure and non-azure.